src/main/webapp/WEB-INF/tags/instructor/home/courseTable.tag

```diff
@@ -48,7 +48,7 @@
                     ${sessionRow.status}
                 </span>
             </td>
-            <td class="session-response-for-test<c:if test="${sessionRow.recent}"> recent</c:if>">
+            <td class="session-response-for-test">
```

Comment:
@damithc that's not any of the present dev's fault (#3601) and changing it here will cause unnecessary troubles.